raven/raven.go

```diff
@@ -70,6 +70,7 @@ type Event struct {
 	ServerName string                 `json:"server_name"`
 	StackTrace stacktrace.StackTrace  `json:"stacktrace"`
 	Http       *Http                  `json:"request"`
+	TargetDsn  string                 `json:"targetDsn"`
```

Comment:
This is just glog-contrib's version of glog event. I just set it here so it could be easily read by the capture loop.
I don't see how we could avoid creating multiple clients.